src/Resources/contao/library/Contao/DcaLoader.php

```diff
@@ -85,7 +85,10 @@ public function load($blnNoCache=false)
 					include $file;
 				}
 			}
-			catch (\InvalidArgumentException $e) {}
+			catch (\InvalidArgumentException $e)
+			{
+				\System::getContainer()->get('monolog.logger.contao')->notice($e->getMessage(), ['exception' => $e]);
```

Comment:
It would log if no such file was found in any `dca` folder, right? Which should theoretically not happen because we would not find the table…

My issue was that an `InvalidArgumentException` was thrown by a DCA file (in this case `PaletteManipulator` but it could be anything), and there was no way to figure out what went wrong and why my file was not loaded.

Maybe we should create a PR for Symfony to throw a `ResourceNotFoundException` in the next version?